src/python_interpreters.rs

```diff
@@ -65,7 +67,7 @@ pub trait CodeObject {
     fn argcount(&self) -> i32;
     fn varnames(&self) -> *mut Self::TupleObject;
 
-    fn get_line_number(&self, lasti: i32, table: &[u8]) -> i32;
+    fn get_line_number(&self, lasti: i32, table: &[u8]) -> Result<i32, Error>;
```

Comment:
is there a reason not to just do this simpler form everywhere? 
```suggestion
    fn get_line_number(&self, lasti: i32, table: &[u8]) -> Result<i32>;
```

https://docs.rs/anyhow/latest/anyhow/#details

Reply:
it's written out like this throughout the rest of the codebase, better to stay consistent